gwm_create_service_account.py

```diff
@@ -305,6 +305,13 @@ async def download_service_account_key():
   await retryable_command(command)
 
 
+async def delete_key():
+  input("\nPress any key to continue...")
+  logging.debug(f"Deleting key file ${KEY_FILE}...")
+  command = f"shred -u {KEY_FILE}"
+  await retryable_command(command)  
+
```

Comment:
Redundant empty line?

Reply:
This empty line is consistent and correct with the rest of the code.